docs/process/sync-testing.md

```diff
@@ -0,0 +1,139 @@
+---
+id: sync-testing
+title: Sync Testing
+sidebar_label: Sync Testing
+---
+
+## Overview
+Testing Sync is complex; it involves testing multiple possible clients, engines, storage options, all against different environments. Configuring each of these can be time consuming; here's a cheat sheet to help you get up and running.
+
+## Testing Against Specific Environments
+Note: these documents are all geared towards the newer [syncstorage-rs](https://github.com/mozilla-services/syncstorage-rs) project. Until the legacy Sync server is retired, you can find detailed instructions on it's usage [here](https://mozilla-services.readthedocs.io/en/latest/howtos/run-sync-1.5.html).
+
+### Development
```

Comment:
I'd like to also document mobile clients for the future in the dev section, but right now am prioritizing getting staging docs up and running.